conf/test_full.config

```diff
@@ -14,11 +14,8 @@ params {
     config_profile_name        = 'Full test profile'
     config_profile_description = 'Full test dataset to check pipeline function'
 
-    // Input data for full size test
-    // TODO nf-core: Specify the paths to your full test data ( on nf-core/test-datasets or directly in repositories, e.g. SRA)
-    // TODO nf-core: Give any required params for the test so that command line flags are not needed
-    input = 'https://raw.githubusercontent.com/nf-core/test-datasets/viralrecon/samplesheet/samplesheet_full_illumina_amplicon.csv'
+    input = 'https://raw.githubusercontent.com/nf-core/test-datasets/molkart/test_data/samplesheets/samplesheet_full_test.csv'
 
-    // Genome references
-    genome = 'R64-1-1'
+    segmentation_method =  "mesmer,cellpose"
+    cellpose_pretrained_model: "nuclei"
```

Comment:
segmentation_method = "mesmer,cellpose" (double space before quotes)

cellpose_pretrained_model = "nuclei" (equals instead of : )